Dockerfile

```diff
@@ -85,9 +85,11 @@ RUN for f in "/etc/passwd" "/projects"; do \
 # Install sonarqube scanner
 RUN wget https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-4.2.0.1873-linux.zip && \
   unzip sonar-scanner-cli-4.2.0.1873-linux.zip  && \
-  mv sonar-scanner-4.2.0.1873-linux/bin/* /usr/local/bin
+
+  mv sonar-scanner-4.2.0.1873-linux /var/opt
+
 # install unzip & buildah
-RUN yum install unzip buildah -y
```

Comment:
@hanzala1234 why did you remove unzip?

Reply:
if buildah could be installed with microdnf I would say to use that one